.github/HOWTO.md

```diff
@@ -0,0 +1,67 @@
+# GitHub Actions
+GitHub Actions help to automate and customize workflows. We deploy Atlas Broker to Cloud Foundry using [GitHub Actions](https://docs.github.com/en/actions). Also, they are used for other tasks like release Atlas-OSB, cleaning our Atlas organization, testing, and demo-runs.
+
+## Using GitHub Actions locally
+Tools for successfully running pipeline locally:
+- `act` allows running GitHub actions without pushing changes to a repository, more information [here](https://github.com/nektos/act)
+- `githubsecrets` helps us to change/create [Github secrets](https://github.com/unfor19/githubsecrets) from CLI
+
+## Req. for running act
+Put the file `.actrc` to the root project folder with used secrets in GitHub
+
+```
+-s DOCKER_PASSWORD=<dd>
+-s DOCKER_USERNAME=<dd>
+-s DOCKER_REPO=<OWNER/IMAGE_NAME>
+```
+
+## Ways to run
+
+Simply call trigger:
+
+```bash
+act push

```

Comment:
[nit] is this the name of a real workflow/job? If not - do you think we could use the real one (is it for workflows only?)

Reply:
this command call push workflow whatever it is